modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testFunctional/tests/ExposeActionsForAvailableIndividualEntitiesSchemasOfHeadlessAdminTaxonomy.testcase

```diff
@@ -0,0 +1,185 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-180090=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Headless";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given in Headless Admin Taxonomy of API Explorer") {
+			APIExplorer.navigateToOpenAPI(
+				api = "headless-admin-taxonomy",
+				version = "v1.0");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCPNoSelenium();
+		}
+	}
+
+	@priority = 4
+	test CanGetMethodAndHrefForKeywordActions {
+		task ("When I navigate to the Keyword Schema") {
+			Click(
```

Comment:
Just wondering if we need to create a related method in macro to expand all actions in Keyword schema, I mean something like this:
```
task ("When I navigate to the Keyword Schema") {
  APIExplorer.expandAllActionsInSchema(schema="Keyword", actionsList="");
}
```
 Let's see what @magjed4289 suggest us to do :smile:

Reply:
I think it's a better way to use a macro to replace and I have modified the PR